src/qibocal/update.py

```diff
@@ -231,3 +240,19 @@ def coupling(g: float, platform: Platform, qubit: QubitId):
 def kernel(kernel: np.ndarray, platform: Platform, qubit: QubitId):
     ro_channel = platform.qubits[qubit].acquisition
     platform.update({f"configs.{ro_channel}.kernel": kernel})
+
+
+def feedback(feedback: list[float], platform: Platform, qubit: QubitId):
+    """Update flux pulse feedback filter parameter in platform for specific qubit."""
+    feedbackQM = feedback.copy()
+    feedbackQM[1] = -feedbackQM[1]
+    platform.update(
+        {f"configs.{platform.qubits[qubit].flux}.filter.feedback": feedbackQM}
+    )
```

Comment:
```suggestion
    platform.update(
        {f"configs.{platform.qubits[qubit].flux}.filter.feedback": -feedbackQM[1]}
    )
```
QM only needs the second term.